packages/page-claims/src/index.tsx

```diff
@@ -73,183 +68,242 @@ const Signature = styled.textarea`
+              <Card withBottomMargin>
+                <h3>{t('2. Enter the ETH address from the sale.')}</h3>
+                <Input
+                  autoFocus
+                  className='full'
+                  help={t('The the Ethereum address you used during the pre-sale (starting by "0x")')}
+                  label={t('Pre-sale ethereum address')}
+                  onChange={onChangeEthereumAddress}
+                  value={ethereumAddress || ''}
+                />
+                {(step === Step.ETHAddress) && (
+                  <Button.Group>
+                    <Button
+                      icon='sign-in'
+                      isDisabled={!ethereumAddress}
+                      label={t('Continue')}
+                      onClick={goToStepSign}
+                    />
+                  </Button.Group>
+                )}
+              </Card>
+            )}
+          {(step >= Step.Sign && !isPreclaimed) && (
+            <Card>
+              {(isOldClaimProcess || !!statementKind)
+                ? <>
+                  <h3>{t('{{step}}. Sign with you ETH address',
+                    { replace: { step: isOldClaimProcess ? '2' : '3' } })}</h3>
+                  {!isOldClaimProcess && (<Statement
+                    kind={statementKind}
+                  />) }
+                  <div>
+                    {t('Copy the following string and sign it with the Ethreum account you used during the pre-sale in the wallet of your choice, using the string as the payload, and then paste the transaction signature object below')}
+                    :
+                  </div>
+                  <CopyToClipboard
+                    onCopy={onCopy}
+                    text={payload}
+                  >
+                    <Payload
+                      data-for='tx-payload'
+                      data-tip
+                    >
+                      {payload}
+                    </Payload>
+                  </CopyToClipboard>
+                  <Tooltip
+                    place='right'
+                    text={didCopy ? t('copied') : t('click to copy')}
+                    trigger='tx-payload'
+                  />
+                  <Signature
+                    onChange={onChangeSignature}
+                    placeholder={`{\n  "address": "0x ...",\n  "msg": "${prefix}:...",\n  "sig": "0x ...",\n  "version": "2"\n}`}
+                    rows={10}
+                  />
+                  {(step === Step.Sign) && (
+                    <Button.Group>
+                      <Button
+                        icon='sign-in'
+                        isDisabled={!accountId || !signature}
+                        label={t('Confirm claim')}
+                        onClick={goToStepClaim}
+                      />
+                    </Button.Group>
+                  )}
+                </>
+                : <div>
+                    The Ethereum address you submitted could not be found in the genesis.
```

Comment:
?

Reply:
Also, since there is one element (which is the fixed `t('...')`), keep it alongside the div, no need to break the line. (As above)